ToDo/src/main/java/com/todo/demo/Dao/NoteDao.java

```diff
@@ -0,0 +1,20 @@
+package com.todo.demo.Dao;
+
+import com.todo.demo.Note.Note;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+public interface NoteDao {
+
+    int saveNote(Note note);
+    List<Note> read();
```

Comment:
call this list notes, and add pagination and sorting

Reply:
Renamed. I will look into the latter ones